catalog/app/containers/NavBar/Help.tsx

```diff
@@ -103,6 +103,11 @@ const syntaxHelpRows: SyntaxHelpNamespace[] = [
         syntax: 'version_id:',
         title: 'Object version id',
       },
+      {
```

Comment:
I'd put `pointer_file` in between `metadata` and `size` to keep alphabetical order
But this is not a strong opinion, because anyway `package_stats.*` breaks this order